groups/wg-productivity/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- productivity-writers
+reviewers:
+- productivity-writers
+- productivity-reviewers
+
+labels:
+- sig/windows
```

Comment:
Knative does not have sig (Special Interest Group), so everywhere that has sig should be deleted.